config/config.properties

```diff
@@ -223,6 +223,15 @@ bookkeeper.zkAddress=master.mesos:2181
 # This value must be the same for all Pravega SegmentStore instances in this cluster.
 #bookkeeper.zkMetadataPath=/segmentstore/containers
 
+# The maximum number of concurrent writes to BookKeeper. Writes are queued and will be executed in order.
+# Recommended values: 1-10. The best value will depend a lot on the installation details and BookKeeper capacity and
+# should be chosen empirically, based on performance tests.
+#bookkeeper.maxConcurrentWrites=5
```

Comment:
Well,  Sure. But I don't know how to automate it or make it dynamic. It should be set based on the performance of the actual installation of BookKeeper, and other settings, such as striping, may affect it. We may be able to make it a function of recent BK performance, but I don't have enough data points yet on how I may be able to do that.